optuna/integration/cma.py

```diff
@@ -459,8 +459,11 @@ def _to_optuna_params(
     ) -> Any:
 
         dist = search_space[param_name]
+        if isinstance(dist, UniformDistribution):
+            # Type of cma_param_value is np.floating, so cast it to Python's built-in float.
+            return float(cma_param_value)
         if isinstance(dist, LogUniformDistribution):
-            return math.exp(cma_param_value)
+            return float(math.exp(cma_param_value))
```

Comment:
Maybe I'm missing something, but `math.exp` returns `float` value if its argument is `numpy.float` value, so `float` cast can be removed:

```suggestion
            return math.exp(cma_param_value)
```

I locally confirmed this as follows and also checked the `tests/samplers_tests/test_samplers.py` was passed.
```bash
python -c "import numpy as np; import math; print(isinstance(math.exp(np.float64(0.)), np.floating))"
False
```

Reply:
You're absolutely right. I fixed it in https://github.com/optuna/optuna/pull/2897/commits/e812161c4047f58c782ce849b101fe71acf0d726. Thank you.